model/src/main/java/io/syndesis/model/Kind.java

```diff
@@ -41,6 +41,9 @@
     Permission(io.syndesis.model.user.Permission.class),
     Role(io.syndesis.model.user.Role.class),
     User(io.syndesis.model.user.User.class),
+
+    TechExtension(io.syndesis.model.techextension.TechExtension.class),
```

Comment:
hey Nicola, I have checked and `io.syndesis.model.extension.Extension` and `io.syndesis.model.techextension.TechExtension` have currently a quite different look, with `TechExtension` having more params. Should I try to replace usage of `Extension` with `TechExtension`?

Reply:
Yes, I left it almost empty because I just needed a container, so the official one is yours. There are also 2 pending PRs that use that extension object (https://github.com/syndesisio/syndesis-rest/pulls/nicolaferraro). So we can fix duplication here or merge all three PRs and then fix the anomaly.